### DISCUSSION TITLE
Auto import jsDoc into stories.mdx

### DISCUSSION THREAD

**Discussion by rexkenley** [storybookjs/storybook#24809] [Help]
We have several ts files that contain functions. The functions are documented using jsDoc. Is it possible to create stories.mdx files that would auto import the functions jsDoc?

e.g.
![image](https://github.com/storybookjs/storybook/assets/10548251/1ca0102d-acd4-4bc9-90de-81d0db78a092)

Thanks!

**Comment by jonniebigodes** [comment]
@rexkenley thanks for reaching out to us with your question. We appreciate it 🙏 ! One thing that I would like to point out is that writing stories using the `.stories.mdx` approach was officially [deprecated](https://storybook.js.org/docs/react/writing-docs/mdx#mdx-and-csf) with the 7.0 release. However, so that you're aware, when Storybook loads the documentation it will automatically infer the jsDoc's comments you have provided, minus some of the tags. If you're ok with it, can you reply in the following [discussion](https://github.com/storybookjs/storybook/discussions/24803) so that the maintainers are aware of it? Or, if you want to collaborate with us on it, might I recommend you open up [RFC](https://github.com/storybookjs/storybook/discussions/new?category=rfc) so that we can follow up with you on it and help you get it implemented? If that's the case, check our [RFC documentation](https://storybook.js.org/docs/react/contribute/RFC) which outlines the whole process for it.

Hope you have a great day.

Stay safe.

**Comment by gluharry** [comment]
My team is also in need of such a feature.
We have several functions that we want to document and have to make workarounds, as Storybook only provides component stories.

It would be lovely to have a way to make function/method stories.

Keep up the great work!
